app/models/CategorisationAnswers.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import cats.data.{EitherNec, NonEmptyChain}
+import cats.implicits.catsSyntaxTuple2Parallel
+import models.AssessmentAnswer.NoExemption
+import models.ott.{CategorisationInfo, CategoryAssessment}
+import org.apache.pekko.Done
+import pages.{AssessmentPage, HasSupplementaryUnitPage, SupplementaryUnitPage}
+import play.api.libs.json.{Json, OFormat}
+import queries.CategorisationQuery
+
+final case class CategorisationAnswers(
+  assessmentValues: Seq[AssessmentAnswer],
+  supplementaryUnit: Option[Int]
+)
+
+object CategorisationAnswers {
+
+  implicit lazy val format: OFormat[CategorisationAnswers] = Json.format
+
+  def build(userAnswers: UserAnswers): EitherNec[ValidationError, CategorisationAnswers] =
+    (
+      getAssessmentAnswers(userAnswers),
+      getSupplementaryUnit(userAnswers)
+    ).parMapN(CategorisationAnswers.apply)
+
+  private def getSupplementaryUnit(userAnswers: UserAnswers) =
+    userAnswers.getOptionalPageValueForOptionalBooleanPage(
+      userAnswers,
+      HasSupplementaryUnitPage,
+      SupplementaryUnitPage
+    )
+
+  private def getAssessmentAnswers(userAnswers: UserAnswers) =
```

Comment:
argh - the function names!
getAssessmentAnswers and getAnsweredAssessments!!!

Reply:
Didn't even notice - sorted